cabot/cabotapp/views.py

```diff
@@ -836,6 +836,7 @@ class ScheduleUpdateView(LoginRequiredMixin, UpdateView):
     context_object_name = 'schedules'
 
     def get_success_url(self):
+        reset_shifts.delay(self.object.id)
```

Comment:
Two comments

1. This is not the right place to call it. Can you please update the PR description with more context? Will help me understand the problem exactly (I am sorry. I know you explained it, but I was so tired yesterday that I forgot. old age!!)
1. `delay` is a deprecated celery api. We should be using `apply_async`